scripts/src/java/org/oppia/android/scripts/ScriptResultConstants.kt

```diff
@@ -0,0 +1,7 @@
+package org.oppia.android.scripts
+
+object ScriptResultConstants {
+  /** RegexPatternValidationCheck output message constants **/
```

Comment:
This is the KDoc for the immediate next element, but it's not describing that constant. This probably should be on the object, or rephrased for the constants.

Reply:
okay.